.github/workflows/all-checks.yml

```diff
@@ -7,6 +7,8 @@ on:
   pull_request:
     branches:
       - main
+  workflow_dispatch:
+  workflow_call:
```

Comment:
Does this do anything?

Reply:
`workflow_dispatch` makes it possible to trigger it manually from the Actions tab and `workflow_call` makes it possible for nightly-build to call this. I could move `workflow_dispatch` from here to nightly build actually!